en/stacks/disk/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Disk management
+---
+
+# Disk management
+
+There are currently two different ways to manage disks and their mount points
```

Comment:
We should get rid of the udisks repo then if it's not being used. That's quite confusing.